cfn-resources/project/mongodb-atlas-project.json

```diff
@@ -19,6 +19,9 @@
                 },
                 "IsSchemaAdvisorEnabled": {
                     "type": "boolean"
+                },
+                "IsExtendedStorageSizesEnabled": {
+                    "type": "boolean"
```

Comment:
this new property is just to create?? does it need to be read only??

Reply:
IsExtendedStorageSizesEnabled is updatable property.